composer.json

```diff
@@ -36,13 +36,14 @@
 
         "novactive/ezseobundle": "^7.0.4",
         "sentry/sentry-symfony": "^4.13",
-        "twig/intl-extra": "^3.8",
+        "twig/twig": "^3.9",
+        "twig/intl-extra": "*",
```

Comment:
PHPStorm will complain when you have unbound dependencies, so if we really need to have both deps listed (which is fine), i'd rather have them synced up, just like symfony packages are synced up.

Reply:
Okay, amended and force pushed.